docs/team/yungweezy.md

```diff
@@ -3,10 +3,60 @@ layout: page
 title: Choon Wei's Project Portfolio Page
 ---
 
-## Project: AddressBook Level 3
+## Project: TutorBuddy
 
-TutorBuddy - Level 3 is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+TutorBuddy is a desktop application made for freelance tutors who give one-to-one tuition to efficiently manage their students’ contacts,
+provide a quick overview of scheduled tuition sessions, and handle monthly tuition fees calculation.
+TutorBuddy is also optimized for fast typing users to handle their day-to-day administrative responsibilities effectively.
 
-Given below are my contributions to the project.
+## Summary of Contributions
 
-* **Code contributed**: [RepoSense link]()
+Given below is a summary of my contributions to the development of TutorBuddy.
+
+### Major Enhancement
+
+* Overlapping Sessions Check
+    - **What it does**: This feature helps to validate the specified session date in the `add_session` and `add_rec_session` command.
+    - **Justification**: This feature provides an important check that informs the tutor if he/she already has a session that falls within the specified date.
+      This prevents the tutor from scheduling two students on the same date and time, reducing the possibilities of awkward session cancellation/postponement.
+    - **Rationale**: The intricacies of this feature was very much underestimated. Everytime `add_session` and `add_rec_session` is called, the check will take place.
```

Comment:
Hmmmm is "Rationale" supposed to be "Highlight"?

Reply:
Ah yes i agree probably highlight

Reply:
Thank you good sir and mdm, I will make the necessary changes :)